openstack/keppel-auto/values.yaml

```diff
@@ -0,0 +1,12 @@
+owner-info:
+  support-group: containers
+  service: keppel
+  maintainers:
+    - Stefan Majewsky
+    - Sandro Jäckel
+    - Stefan Voigt
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/openstack/keppel-auto
+
+# See docstring on keppel-managed-accounts.yaml in the secrets repo for details.
```

Comment:
Can't we link the keppel doc instead?

Reply:
No, the format of the `accounts` section deviates in some ways from the config file that Keppel sees, because it is written in a region-agnostic way.